core/silkworm/common/lru_cache.hpp

```diff
@@ -80,6 +81,12 @@ class lru_cache {
         }
     }
 
+    std::optional<value_t> get_a_copy(const key_t& key) {
```

Comment:
Probably better `get_as_copy` instead of `get_a_copy`

Reply:
done